ospfd/ospf_vty.c

```diff
@@ -11263,7 +11259,9 @@ DEFUN (show_ip_ospf_route,
 		ret = show_ip_ospf_route_common(vty, ospf, json, use_vrf);
 		/* Keep Non-pretty format */
 		if (uj)
-			vty_out(vty, "%s\n", json_object_to_json_string(json));
+			vty_out(vty, "%s\n",
+				json_object_to_json_string_ext(
+					json, JSON_C_TO_STRING_NOSLASHESCAPE));
```

Comment:
Can't we print PRETTY everywhere?

Reply:
pretty can add a significant amount of overhead to output time.  We must make a decision that this is ok to use or not depending on expected amount of json dumped

Reply:
I'm okay with this not being PRETTY ... we should discuss when we should and shouldn't PRETTY at some point I reckon.